### PR TITLE
II-94-collection-item-test

### PR DESCRIPTION
**Pull request by jinyoungkim927**
[Jira ticket](https://immutable.atlassian.net/jira/software/projects/II/boards/294?selectedIssue=II-94)

E2E test:
- User lands on home / collection page
- User clicks a collection
- Collection item displays as a result

### CODE REVIEW

File: cypress/cypress.config.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const { defineConfig } = require('cypress')
```
Comment (aerim-yi) [inline cypress/cypress.config.js:1]: We already have a config file in the main branch. 
Check out `cypress/cypress.config.ts`

File: cypress/e2e/collection.spec.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// Test for clicking collection item from home page 
+
+describe('Clicking into a collection ', () => {
+it('finds the content "type"', () => {
+    cy.visit("http://localhost:3000");
```
Comment (aerim-yi) [inline cypress/e2e/collection.spec.js:5]: We set our baseUrl as https://intern-x-marketplace.herokuapp.com/ in the `cypress.config.ts` file. 
```suggestion
    cy.visit("/");
```

File: cypress/cypress.config.js (hunk 1 above)
Comment (jinyoungkim927) [reply cypress/cypress.config.js:1]: Yep I'll delete it

**Comment by jinyoungkim927** [comment]
Closed PR as it could cause version control issues to the rest of the codebase. Added PR contents to II-96

**Pull request closed without merging**